eng/Versions.props

```diff
@@ -55,7 +55,12 @@
     <MicrosoftDotNetGenAPIPackageVersion>1.0.0-beta.19302.2</MicrosoftDotNetGenAPIPackageVersion>
     <!-- Packages from dotnet/core-setup -->
     <MicrosoftExtensionsDependencyModelPackageVersion>3.0.0-preview7-27810-02</MicrosoftExtensionsDependencyModelPackageVersion>
-    <MicrosoftNETCoreAppPackageVersion>3.0.0-preview7-27810-02</MicrosoftNETCoreAppPackageVersion>
+    <MicrosoftNETCoreAppRefPackageVersion>3.0.0-preview7-27810-02</MicrosoftNETCoreAppRefPackageVersion>
+    <!--
+      Win-x64 is used here because we have picked an arbitrary runtime identifier to flow the version of the latest NETCore.App runtime.
+      All Runtime.$rid packages should have the same version.
+    -->
+    <MicrosoftNETCoreAppRuntimewinx64PackageVersion>3.0.0-preview7-27810-02</MicrosoftNETCoreAppRuntimewinx64PackageVersion>
```

Comment:
Thought of this after I merged extensions. Maybe it would be better if we made the code reflect this intention better by adding this mapping:
```xml
<MicrosoftNETCoreAppRuntimeVersion>$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)</MicrosoftNETCoreAppRuntimeVersion>
```

Seeing how widespread the use of the MicrosoftNETCoreAppRuntimewinx64PackageVersion variable is makes me worry this will get confused by others reading the code.

Reply:
I was confused ;) but not much confuses me

Reply:
`<MicrosoftNETCoreAppRuntimePackageVersion>$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)</MicrosoftNETCoreAppRuntimePackageVersion>`

Reply:
I agree you should have another variable, even if it's initialized with the winx64 value

Reply:
https://github.com/aspnet/Extensions/pull/1832

Reply:
hm 
![image](https://user-images.githubusercontent.com/8302101/59387900-37bac380-8d1f-11e9-9866-97cd55a77b96.png)